website/ref/index.toml

```diff
@@ -10,6 +10,10 @@ title = "The Builtin Module"
 name = "edit"
 title = "The Editor Module (edit:)"
 
+[[articles]]
+name = "platform"
+title = "Access to the platform’s identifying data (platform:)"
```

Comment:
We do not use the typographically correct apostrophe `’` elsewhere in the website, so stick to `'`.